apps/api/src/services/form.service.ts

```diff
@@ -23,6 +23,72 @@ export class FormService {
 		return await FormModel.getAllFormsWithSubmissionsCount(eventId);
 	}
 
+	async deleteFormField(formId: string, fieldId: string) {
+		try {
+			const formFieldInfo = await FormFieldsModel.findOne({
+				id: fieldId,
+				fk_form_id: formId,
+			});
+			if (!formFieldInfo) {
+				throw FFError.notFound(
+					`${SystemTable.FormFields}: Query Failed : 
+          form with id ${formId} could not be found`,
+				);
+			}
+
+			await FormFieldsModel.delete({
+				id: fieldId,
+			});
+
+			return {
+				ok: true,
+				message: 'Field deleted successfully',
+			};
+		} catch {
+			throw new InternalServerErrorException();
+		}
```

Comment:
Don't catch and throw like this. I noticed this pattern in many places.  If error happens, it is thrown inside the delete functions and captured by exception filter.